clarifai_grpc/channel/http_client.py

```diff
@@ -120,6 +120,10 @@ def _encode_get_params(self, params):
       elif isinstance(v, dict):
         for (subk, subv) in self._encode_get_params(v).items():
           encoded_params[k + '.' + subk] = subv
+      elif isinstance(v, list):
+        text_list = [str(x) for x in v]
```

Comment:
why is it always a str()?

Reply:
oops. forgot to push my change.
I saw this in our repo and had the same question.